vector/src/main/java/im/vector/app/features/call/VectorCallActivity.kt

```diff
@@ -636,18 +639,36 @@ class VectorCallActivity : VectorBaseActivity<ActivityCallBinding>(), CallContro
 
     private val screenSharingPermissionActivityResultLauncher = registerStartForActivityResult { activityResult ->
         if (activityResult.resultCode == Activity.RESULT_OK) {
-            callViewModel.handle(VectorCallViewActions.StartScreenSharing)
-            // We need to start a foreground service with a sticky notification during screen sharing
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
-                ContextCompat.startForegroundService(
-                        this,
-                        Intent(this, ScreenCaptureService::class.java)
-                )
-                screenCaptureServiceConnection.bind()
+                // We need to start a foreground service with a sticky notification during screen sharing
+                startScreenSharingService(activityResult)
+            } else {
+                startScreenSharing(activityResult)
             }
         }
     }
 
+    private fun startScreenSharing(activityResult: ActivityResult) {
+        val videoCapturer = ScreenCapturerAndroid(activityResult.data, object : MediaProjection.Callback() {
+            override fun onStop() {
+                Timber.v("User revoked the screen capturing permission")
```

Comment:
If we're keeping this in production I would make this `Timber.i`

Reply:
Done. Yes, we have to implement this dummy callback.